webapp/src/components/App/Studies/LauncherDialog.tsx

```diff
@@ -170,12 +172,17 @@ function LauncherDialog(props: Props) {
   // Utils
   ////////////////////////////////////////////////////////////////
 
-  const timeLimitParse = (value: string): number => {
-    try {
-      return parseInt(value, 10) * 3600;
-    } catch {
-      return 48 * 3600;
-    }
+  /**
+   * Parses an hour value from a string and converts it to seconds.
+   * If the input is invalid, returns a default value (240 hours).
+   *
+   * @param hourString - A string representing the number of hours.
+   * @returns The equivalent number of seconds, or a default value for invalid inputs.
+   */
+  const parseHoursToSeconds = (hourString: string): number => {
+    const hours = parseInt(hourString, 10);
```

Comment:
```typescript
const seconds = moment.duration(hourString, 'hours').asSeconds();
return seconds > 0 ? seconds : DEFAULT_TIME_LIMIT;
```

Reply:
I think its overkill to use moment for this, but thanks for the alternative

Reply:
Is aleready imported, so no problem

Reply:
Not only about bundle size, JS native parse is faster than moment and I don't see the point using an external lib for such simple thing. But let's add it for readability if you prefer this, it's a bit more simple